evaluator/src/main/kotlin/Evaluator.kt

```diff
@@ -22,15 +22,18 @@ package com.here.ort.evaluator
 import com.here.ort.model.EvaluatorRun
 import com.here.ort.model.OrtResult
 import com.here.ort.model.RuleViolation
+import com.here.ort.model.licenses.LicenseConfiguration
 import com.here.ort.utils.ScriptRunner
 
 import java.time.Instant
 
-class Evaluator(ortResult: OrtResult) : ScriptRunner() {
+class Evaluator(ortResult: OrtResult, licenseConfiguration: LicenseConfiguration = LicenseConfiguration()) :
+    ScriptRunner() {
```

Comment:
If the signature does not fit on a single line, use one line per parameter, because "If we use inheritance, then the superclass constructor call or list of implemented interfaces should be located on the same line as the parenthesis". https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

Reply:
Interresting thanks! Done. Even the auto formater does not seem to know about this one.